src/allotropy/parsers/unchained_labs_lunatic/unchained_labs_lunatic_reader.py

```diff
@@ -0,0 +1,12 @@
+import numpy as np
+import pandas as pd
+
+from allotropy.types import IOType
+
+
+class UnchainedLabsLunaticReader:
+    @staticmethod
+    def read(contents: IOType) -> pd.DataFrame:
+        data: pd.DataFrame = pd.read_csv(filepath_or_buffer=contents)
```

Comment:
I think you can drop this class and pull this into the parser function that uses it.